src/shared/components/palette/case-flag/components/case-flag-table/case-flag-table.component.html

```diff
@@ -9,8 +9,8 @@
       <th scope="col" class="govuk-table__header col-flag-status">Flag status</th>
     </tr>
   </thead>
-  <tbody class="govuk-table__body" *ngIf="!flagData.details || (flagData.details && flagData.details.length === 0)">
-    <tr>
+  <tbody class="govuk-table__body">
+    <tr *ngIf="!flagData.details || (flagData.details && flagData.details.length === 0)">
```

Comment:
@johnbenjamin-hmcts - I've asked @ronaldmansveld for his input on this one... not sure whether it's better to remove the `<tbody>` altogether if it has no rows (hence the `*ngIf` being at this level originally), or leave a potentially empty `<tbody>` element

Reply:
@johnbenjamin-hmcts - @ronaldmansveld's response:

> Leave the empty `tbody`
> as that indicates that there are no rows
> and you still want the `thead` for the headers
> and believe `tbody` is required when you use `thead`

On that basis, I'm resolving this comment